server/plugin/utils.go

```diff
@@ -206,3 +206,41 @@ func (p *Plugin) IsAnyProjectLinked(mattermostUserID string) (bool, error) {
 func (p *Plugin) getConnectAccountFirstMessage() string {
 	return fmt.Sprintf(constants.ConnectAccountFirst, fmt.Sprintf(constants.ConnectAccount, p.GetPluginURLPath(), constants.PathOAuthConnect))
 }
+
+func (p *Plugin) ParseSubscriptionsToCommandResponse(subscriptionsList []serializers.SubscriptionDetails, channelID string) string {
+	var sb strings.Builder
+
+	if len(subscriptionsList) <= 0 {
+		sb.WriteString(constants.NoSubscriptionFound)
+		return sb.String()
+	}
+
+	sb.WriteString("###### Board subscription(s) for this channel\n")
+	sb.WriteString("| Subscription ID | Organization | Project | Event Type |\n")
+	sb.WriteString("| :-------------- | :----------- | :------ | :--------- |\n")
+
+	noSubscriptionFound := true
```

Comment:
why can we have a variable named subscriptionFound and do this exactly opposite of this

Reply:
that will need to add extra check as `!subscriptionFound` which is unnecessary